### PR TITLE
{lib}[GCCcore/11.2.0] c-ares v1.18.1

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node302.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/367138f188f0ff62c2fdb56a383685d4 for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
software2.lisa.surfsara.nl - Linux debian 10.11, x86_64, Intel(R) Xeon(R) Bronze 3104 CPU @ 1.70GHz, 4 x NVIDIA NVIDIA TITAN V, 470.103.01, Python 2.7.16
See https://gist.github.com/8d4d9b2ee0184f978af54bf6756b64f0 for a full test report.

File: easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+source_urls = [GITHUB_SOURCE]
+sources = ['%s-%s.tar.gz' % (name.replace('-', ''), version.replace('.', '_'))]
+checksums = ['414872549eec4e221b576693fdc9c9bce44ff794d0f1f06f2515b56a7f6ec9c9']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('Autotools', '20210726'),
+]
+
+preconfigopts = 'autoreconf -i -f &&'
```
Comment (casparvl) [inline easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCCcore-11.2.0.eb:22]: I see that this one is new compared to `c-ares-1.17.2-GCCcore-10.3.0.eb`. Any particular reason for it? Do you run into errors without the `autoreconf`?

Comment (lexming) [reply easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCCcore-11.2.0.eb:22]: The sources from github only ship `configure.ac`. So this is needed to generate the `configure` script.

Comment (casparvl) [reply easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCCcore-11.2.0.eb:22]: Ok, pretty good reason then, thanks for letting me know ;-)

**Review by casparvl — APPROVED** [review]
LGTM!

**Comment by casparvl** [comment]
Going in, thanks @lexming!

**Pull request merged**